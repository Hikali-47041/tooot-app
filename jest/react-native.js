jest.mock('react-native/Libraries/LayoutAnimation/LayoutAnimation', () => ({
  ...require.requireActual(
    'react-native/Libraries/LayoutAnimation/LayoutAnimation'
  ),
  configureNext: jest.fn()
}))
