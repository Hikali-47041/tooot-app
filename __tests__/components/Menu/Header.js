import React from 'react'
import { cleanup, render } from '@testing-library/react-native/pure'

import MenuHeader from '@components/Menu/Header'

describe('Testing component menu header', () => {
  afterEach(cleanup)

  it('with text only', () => {
    const { getByText, toJSON } = render(<MenuHeader heading='test' />)

    getByText('test')
    expect(toJSON()).toMatchSnapshot()
  })
})
