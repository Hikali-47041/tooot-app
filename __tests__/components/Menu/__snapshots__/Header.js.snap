// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing component menu header with text only 1`] = `
<View
  style={
    Object {
      "paddingBottom": 8,
      "paddingLeft": 16,
      "paddingRight": 16,
    }
  }
>
  <Text
    style={
      Array [
        Object {
          "fontSize": 14,
          "fontWeight": "600",
          "lineHeight": 20,
        },
        Object {
          "color": "rgb(135, 135, 135)",
        },
      ]
    }
  >
    test
  </Text>
</View>
`;
